spark/streamingGuide1/src/main/scala/WordCount.scala

```diff
@@ -0,0 +1,57 @@
+/**
+  * Created by abgoswam on 3/8/17.
+  */
+
+import org.apache.spark._
+import org.apache.spark.streaming._
+import org.apache.spark.sql.SparkSession
+
+object WordCount {
+  def main(args: Array[String]): Unit = {
+
+    // Create a local StreamingContext with two working thread and batch interval of 1 second.
+    // The master requires 2 cores to prevent from a starvation scenario.
+
+    val conf = new SparkConf().setMaster("local[2]").setAppName("WordCount")
+    val ssc = new StreamingContext(conf, Seconds(10))
+
+    // Create a DStream that will connect to hostname:port, like localhost:9999
+    val lines = ssc.socketTextStream("localhost", 9999)
+    // lines.print()
+
+    // Split each line into words
+    val words = lines.flatMap(_.split(" "))
```

Comment:
u can ignore this file.  its one of the 'sample word count' examples I was playing with to understand how to convert a DStream to DataFrames